Gopkg.toml

```diff
@@ -60,7 +60,7 @@
 
 [[constraint]]
   name = "github.com/kata-containers/agent"
-  revision = "a93071539feee29bfa22b6184380d3fd7c156ef7"
+  revision = "c8199f60759a1d52932df37a1af710cdeada8c81"
```

Comment:
Please add the shortlog diff since the last vendoring of this dependency into your commit message. It helps tracking which patches are pulled through this vendoring.

Reply:
Hey, sorry to be picky here, but we don't really need the merge commits. If you run the following command:
```bash
git log --no-merges --abbrev-commit --pretty=oneline a93071539feee29bfa22b6184380d3fd7c156ef7..origin/master
```
You should only get the __real__ commits.

Reply:
Taking notes and commit message updated. Thanks @sboeuf!